CMakeLists.txt

```diff
@@ -90,20 +87,229 @@ if("${CMAKE_SOURCE_DIR}" STREQUAL "${CMAKE_BINARY_DIR}")
     )
 endif()
 
-option(BUILD_HELICS_TESTS "Enable the test Executables to be built" ON)
```

Comment:
We're doing enable_testing based on other things, BUILD_HELICS_TESTS isn't actually propogated

Reply:
This should be removed, I think it is a relic of something when I synchronized with the HELICS code